src/main/java/com/bina/varsim/util/VCFparser.java

```diff
@@ -222,10 +219,14 @@ boolean splitGeno(String geno, byte[] vals, ChrString chr) {
     /**
      * takes a line from a VCF file, parse it,
      * return a Variant object
+     *
+     * right now the meta-info lines (beginning with ##) are
+     * not tied with data line parsing. This will be corrected
+     * in the future (perhaps with help of HTSJDK).
      * @param line
      * @return
      */
-    public Variant process_line(String line) {
+    public Variant process_line(String line) throws UnexpectedException {
```

Comment:
Should switch to consistent naming of functions and variables. Should go with camelcase for variables, classes, functions.